client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -205,6 +218,17 @@ class StatsPostDetail extends Component {
 			titleLogo: null,
 		};
 
+		const subscriptionsEnabled = isSimple || isSubscriptionsModuleActive;
```

Comment:
Code here is mostly moved from https://github.com/Automattic/wp-calypso/pull/103141/files#diff-fc93fdb828c96efc52107151a3c2dff45c20e0702e831cabeed908735a856632L56